__tests__/app.test.js

```diff
@@ -266,6 +266,163 @@ describe("POST /api/articles/:article_id/comments", () => {
   });
 });
 
+describe("PATCH /api/articles/:article_id", () => {
+  test("STATUS 200: returns an updated article specified by article_id (increment votes by provided inc_votes in the body of request)", () => {
+    const newVote = 1;
+    const body = { inc_votes: newVote };
+
+    const expectedOutput = {
+      article_id: 4,
```

Comment:
We should by this point, have already tested for the structure of our articles. It would be enough in this test just to check that the vote property has been updated

Reply:
thanks! fixed!